packages/attestation-service/src/requestHandlers/liveness.ts

```diff
@@ -9,10 +9,14 @@ export async function handleLivenessRequest(_req: express.Request, res: express.
       return
     }
 
+    if (await isNodeSyncing()) {
+      respondWithError(res, 504, ErrorMessages.NODE_IS_SYNCING)
+    }
+
     try {
       await isDBOnline()
```

Comment:
is this where we wanted to handle misconfigured db?  eg. check attestation table exists

Reply:
I may do this in a separate PR